src/gui/input/AxisConfigWidget.cc

```diff
@@ -31,10 +31,9 @@
 #include "QSettingsCached.h"
 #include "input/InputDriverManager.h"
 #include "SettingsWriter.h"
-#include "WheelIgnorer.h"
+#include "../IgnoreWheelWhenNotFocused.h"
```

Comment:
Is the `../` needed? There should be a `-I/openscad/src/gui` to bring in the file without explicit path.